agent/internal/containers.go

```diff
@@ -66,6 +66,15 @@ func (c *containerManager) Receive(ctx *actor.Context) error {
 			fmt.Sprintf("DET_MASTER_PORT=%d", masterPort),
 			fmt.Sprintf("DET_AGENT_ID=%s", c.Options.AgentID),
 		}
+
+		// The master might be listening over both TLS and non-TLS, in which case it will send out
+		// TLS-related fields to all tasks, since it doesn't keep track of which port each agent used. But
+		// if this agent is using a non-TLS connection, we have to make sure tasks do as well, since we
```

Comment:
non-blocking: "If this agent..."

Reply:
The sentence actually starts with the "But" on the previous line. :wink:

Reply:
You should know I can't read by now.